roles/_common/tasks/install.yml

```diff
@@ -102,6 +102,24 @@
       check_mode: false
       when: __common_binary_basename is search('\.zip$|\.tar\.gz$')
 
+- name: "Check existence of binary install dir"
+  ansible.builtin.stat:
+    path: "{{ _common_binary_install_dir }}"
+  register: __common_binary_install_dir
```

Comment:
I wrote this with reference to below format:
https://github.com/prometheus-community/ansible/blob/f3514a9e97dfc3d64f02aefd299487efd99e779f/roles/systemd_exporter/tasks/preflight.yml#L42-L50